packages/apollo-links/src/TokenRefreshLink.js

```diff
@@ -15,7 +15,7 @@ import type {
   RefreshTokenQueryInput,
   RefreshTokenQueryResult,
 } from './types';
-import { hasTokenExpiredError, hasTokenInvalidError } from './utils';
+import { hasIdTokenExpiredError, hasRefreshTokenExpiredError } from './utils';
```

Comment:
Maybe you should move these function to utils package and use all across other packages?